Classes/NSString+Typeset.h

```diff
@@ -10,6 +10,8 @@
 
 @class TypesetKit;
 
+#define Typeset(__X__) (__X__?__X__.typeset:@"".typeset)
```

Comment:
Mmmm. Maybe this comes from a fork merge, but this code is already in your repo.